packages/cli/src/services/user.service.ts

```diff
@@ -0,0 +1,61 @@
+import { Service } from 'typedi';
+import type { EntityManager, FindManyOptions, FindOneOptions, FindOptionsWhere } from 'typeorm';
+import { In } from 'typeorm';
+import { User } from '@db/entities/User';
+import type { IUserSettings } from 'n8n-workflow';
+import { UserRepository } from '@/databases/repositories';
+
+@Service()
+export class UserService {
```

Comment:
We should eventually move more of user-related business-logic into this service, otherwise this file doesn't serve much of a purpose.

For starters, we could move most of the post-input-validation code from `UsersController`, `MeController`, and maybe `PasswordResetController` in here (or similar service classes), use this service in `PublicApi/v1/handlers/users/users.handler.ee.ts`, and delete the `PublicApi/v1/handlers/users/users.service.ee.ts`.

Reply:
Agree, this is an initial setup :) 

Let's keep merging progress in small separate PRs - easier to review and less likely to accumulate conflicts.